src/System.Text.Json/tests/JsonObjectTests.cs

```diff
@@ -113,15 +113,15 @@ public static void TestGuid()
         public static void TestDateTime(DateTime dateTime)
         {
             var jsonObject = new JsonObject { { "dateTime", dateTime } };
-            Assert.Equal(dateTime.ToString("s", CultureInfo.InvariantCulture), ((JsonString)jsonObject["dateTime"]).Value);
+            Assert.Equal(dateTime.ToString("O", CultureInfo.InvariantCulture).Replace(".0000000", string.Empty), ((JsonString)jsonObject["dateTime"]).Value);
```

Comment:
This accounts for the trimming done by `JsonWriterHelper.TrimDateTimeOffset`. Got any suggestions as to how that should be expressed?

Reply:
@khellang @ahsonkhan I'd adapt the writer's trimming tests: https://github.com/dotnet/corefx/blob/7cae92b39d2fed2e4e88e900f2e7d787ed9f6cfa/src/System.Text.Json/tests/Utf8JsonWriterTests.cs#L6147-L6159

Reply:
Thanks! Updated to use the `Utf8JsonWriterTests` pattern 👌